scripts/lib/api/processHtml.test.ts

```diff
@@ -432,14 +432,34 @@ describe("prepareGitHubLink()", () => {
     const doc = Doc.load(
       `<span class="pre">None</span><span class="sig-paren">)</span><a class="reference external" href="https://github.com/Qiskit/qiskit/blob/stable/1.0/qiskit/utils/deprecation.py#L24-L101"><span class="viewcode-link"><span class="pre">[source]</span></span></a><a class="headerlink" href="#qiskit_ibm_runtime.IBMBackend" title="Link to this definition">#</a>`,
     );
-    const result = prepareGitHubLink(doc.$, doc.$main);
+    const result = prepareGitHubLink(doc.$main, false);
     expect(result).toEqual(
-      `<a href="https://github.com/Qiskit/qiskit/blob/stable/1.0/qiskit/utils/deprecation.py#L24-L101" title="view source code">GitHub</a>`,
+      ` <a href="https://github.com/Qiskit/qiskit/blob/stable/1.0/qiskit/utils/deprecation.py#L24-L101" title="view source code">GitHub</a>`,
```

Comment:
Is the extra space important? If so maybe a quick comment so we don't delete it at some point in the future

Reply:
It is https://github.com/Qiskit/documentation/pull/915. I moved where the space is added as a refactor of #915.